docusaurus/docs/plugins/plugins-getting-started.md

```diff
@@ -104,148 +96,90 @@ yarn dev
-
-// Excludes the following plugins if there's no .env file.
-const excludes = process.env.EXCLUDES_PKG || 'epinio, rancher-components, testplugin';
+When we run `yarn dev` at the moment, our test extension will be automatically loaded in the application - this allows us to develop
+the extension with hot-reload.
 
-export default config(__dirname, {
-  excludes: excludes.replace(/\s/g, '').split(','),
-  // excludes: ['fleet', 'example']
-  // autoLoad: ['fleet', 'example']
-});
-```
+To test loading the extension dynamically, we can update configuration to tell Rancher not to include our extension.
 
-**NOTE**: If you're actively working with Plugins, we can make use of the `.env` file instead of hardcoded the excludes in the `nuxt.config.js` file so your changes on the file won't be tracked by Git.
+To do this, create a new `.env` file in the root `test-app` folder, and add these contents:
 
 ```
-# First, create the .env file from template
-cp env.example .env
-
-# Then, add the following to the .env file
-EXCLUDES_PKG='epinio, rancher-components, testplugin'
+EXCLUDES_PKG='test'
 ```
 
-What we have done here is add the `testplugin` plugin to the `excludes` configuration - this means that when we build and run the UI, we won't include the `testplugin` plugin.
+If necessary, bring in the environment variables by running `source .env`.
 
-Run the UI with:
+Now, run the UI with:
 
 ```
 yarn dev
 ```
 
-Open a web browser to `https://127.0.0.1:8005` and you'll see that the Example nav item is not present - since the plugin was not loaded.
+Open a web browser to `https://127.0.0.1:8005` and you'll see that the Example nav item is not present - since the extension was not loaded.
 
-Bring in the slide-in menu (click on the hamburger menu in the top-left) and click on 'Plugins'.
+> Note: You need to be an admin user to test Extensions in the Rancher UI
 
-In the top input box, enter `testplugin` as the Plugin name and click 'Load Plugin' - you should see a notification telling you the plugin was loaded and if you bring in the side menu again, you should see the Example nav item there now.
+Go to the user avatar in the top-right and go to 'Preferences'. Under 'Advanced Features', check the `Enable Extension developer features' checkbox.
 
-This illustrates dynamically loading a Plugin. Note that when we started the UI, it serves up any plugins in the `dist-pkg` folder under the `/pkg` route of the app - so when we entered the name `testplugin` as the plugin, the UI loaded the plugin from:
+Now, bring in the slide-in menu (click on the hamburger menu in the top-left) and click on 'Extensions'.
 
-```
-https://127.0.0.1:8005/pkg/testplugin/testplugin.umd.min.js
-```
+Go to the three dot menu and select 'Developer load' - you'll get a dialog allowing you to load the extension into the UI.
 
-If you copy and paste this URL into a browser, you'll see the Javascript for the plugin.
+In the top input box `Extension URL`, enter:
```

Comment:
is the pkg folder served up alongside the dashboard i.e. no serve-pkgs needed?

Reply:
Yep. Brought out the note on this